_install_guide/adding_accounts.md

```diff
@@ -87,3 +87,114 @@ correct trust policy in IAM.  Below is the trust policy you give the `SpinnakerM
 Below is the latest EC2 policy to use for allowing `SpinnakerInstanceProfile`.
 
 <script src="http://gist-it.appspot.com/https://github.com/Armory/spinnaker-aws-policy/blob/master/policies/latest/SpinnakerInstanceProfile.json"></script>
+
+
+# Adding Artifact Accounts
+
+Artifacts are a main driver behind the Kubernetes V2 provider. They allow you to store Kubernetes manifests in external repositories such as Github or S3 and then deploy those artifacts with Spinnaker. There are many different artifact providers and they are all configured in a similar fashion. 
+
+To configure an artifact account, add any of the following snippets to `clouddriver-local.yml` under the top level `artifacts` key. For example, if you wanted to configure 2 artifact accounts, S3 and Github, you would configure them as follows:
+
+```
+# clouddriver-local.yml
+artifacts:
+  s3:
+    enabled: true
+    accounts:
+      - name: s3
+        region: us-west-2
+  github:
+    enabled: true
+    accounts:
+      - name: github
+        token: personalAccessToken
+```
+
+Below are example configuration for all available artifact providers.
+
+## Bitbucket
+
+```
+artifacts:
+  bitbucket:
+    enabled: true
+    accounts:
+      - name: bitbucket-account-name
+        username: armory-bot
+        password: supersecretpassword
+        # usernamePasswordFile should only be used if username and password are not used
```

Comment:
what does the `usernamePasswordFile` look like?
`armory-bot: supersecretpassword`?

could you put that in a above section?

Reply:
i added it.